tests/utils/kube/status.go

```diff
@@ -119,7 +119,7 @@ func WaitForAnyPodReady(
 		tc:        tc,
 		fetchOpts: fo,
 	}
-	pp.Cond = func() (bool, error) { return pp.checkReady(ctx) }
+	pp.Prober.Cond = func() (bool, error) { return pp.checkReady(ctx) }
```

Comment:
Has the type changed with the new go version or was this a bug all along?

Reply:
It is not a bug, but rather a picky linter. I could not find out why in go1.18 they prefer this style.